### PR TITLE
PLATFORM-1250 Hook Helios' clearAccessTokenCookie() to UserLogout

### PR DESCRIPTION
**Pull request by ghost**
Calling Helios' methods when Helios is disabled (Uncyclo) results with fatal errors which is why it is better to hook to core events rather than make explicit calls.
- extensions/wikia/Helios/Helios.setup.php - hook Wikia\Helios\User::clearAccessTokenCookie() to UserLogout event
- extensions/wikia/Helios/Helios.setup.php - Wikia\Helios\User::clearAccessTokenCookie() returns true so that wfRunHooks('UserLogout') evalates to true, too.
- includes/User.php - remove explicit call from doLogout() and rely on wfRunHooks('UserLogout') in logout()

### CODE REVIEW

File: extensions/wikia/Helios/Helios.setup.php [hunk 1]
```diff
@@ -35,3 +35,4 @@
 
 $wgHooks['UserSaveSettings'][] = 'Wikia\\Helios\\User::onUserSave';
 $wgHooks['UserSaveOptions'][] = 'Wikia\\Helios\\User::onUserSave';
+$wgHooks['UserLogout'][] = 'Wikia\\Helios\\User::clearAccessTokenCookie';
```
Comment (macbre) [inline extensions/wikia/Helios/Helios.setup.php]: Is `Wikia\\Helios\\User::clearAccessTokenCookie` method called only via this hook? if so, let's rename it to `onUserLogout` to match the methods above.

**Comment by michalroszka** [comment]
@garthwebb - per this is per https://github.com/Wikia/app/commit/9da934006954257a131634a7f9770921056df532 and https://wikia-inc.atlassian.net/browse/PLATFORM-1250

File: extensions/wikia/Helios/Helios.setup.php (hunk 1 above)
Comment (michalroszka) [inline extensions/wikia/Helios/Helios.setup.php]: Currently it is, although it seems the author's intention was to make a general purpose method.

**Comment by garthwebb** [comment]
LGTM

**Pull request merged**